draft-looker-oauth-attested-key-based-client-authentication.md

```diff
@@ -94,34 +96,45 @@ The following diagram depicts the conceptual interactions.
                      +------->|               |<------------------------>|               |
   (1) Generate       |        |    Client     |    (6) Interaction       | Authorization |
   Attestation Key    |        |   Instance    |    using Client          |    Server     |
-                     +--------|               |    Key Attestation + PoP |               |
+                     +--------|               |    Attestation + PoP     |               |
                               |               |    for authentication    |               |
                               +---------------+                          +---------------+
                                  / \      |
                                   |       |
                                   +-------+
                                 (5) Generate
-                              Key Attestation PoP
+                           Client Attestation PoP
 ~~~
 
-Note defining the protocol for steps 2 and 4, including how the client instance authenticates with the client backend is out of scope of this specification.
+Note that the data and protocols used for requesting and sending a Client Attestation JWT (steps 2 and 4) and how the client instance authenticates with the client backend is out of scope of this specification.
```

Comment:
Still prefer the previous language to be honest, data exchanged is a part of the protocol IMO and its one protocol not multiple hence protocol instead of protocols. The diagram also already describes what is happening so the "request and sending a client attestation JWT" feels redundant.